src/porepy/fracs/fracture_network_2d.py

```diff
@@ -506,9 +506,9 @@ def _find_and_split_intersections(self, constraints: np.ndarray) -> None:
 
         tags = np.zeros((2, edges.shape[1]), dtype=int)
 
-        tags[0][
-            np.logical_not(self.tags["boundary"])
-        ] = GmshInterfaceTags.FRACTURE.value
+        tags[0][np.logical_not(self.tags["boundary"])] = (
```

Comment:
This is much more readable!

Reply:
Agree. It is strange black did not do that, but to me, sensible auto-formatting is a miracle.